apps/src/netsim/NetSimRouterNode.js

```diff
@@ -689,16 +681,34 @@ NetSimRouterNode.prototype.getStatus = function () {
   if (levelConfig.broadcastMode) {
     return i18n.roomStatus({
       connectedClients: connectedNodeNames,
-      remainingSpace: (MAX_CLIENT_CONNECTIONS - incomingWireRows.length)
+      remainingSpace: (this.maxClientConnections_ - connectionCount)
     });
   }
 
   return i18n.routerStatus({
     connectedClients: connectedNodeNames,
-    remainingSpace: (MAX_CLIENT_CONNECTIONS - incomingWireRows.length)
+    remainingSpace: (this.maxClientConnections_ - connectionCount)
   });
 };
 
+/**
+ * @returns {string[]} the names of all the nodes connected to this router.
+ * @private
+ */
+NetSimRouterNode.prototype.getConnectedNodeNames_ = function () {
+  var cachedNodeRows = this.shard_.nodeTable.readAll();
+  return this.getConnections()
+      .map(function (wire) {
```

Comment:
I'd much prefer to see `return this.getConnections().map(function (wire) {` all on one line here; makes it much harder to accidentally see `return this.getConnections()`  and assume that's all that's being returned.